xilem/examples/calc.rs

```diff
@@ -0,0 +1,326 @@
+// Copyright 2024 the Xilem Authors
```

Comment:
What should be done regarding this? The authors file is for major contributors. This is just an example.